lib/trino-filesystem-azure/src/main/java/io/trino/filesystem/azure/AzureFileSystem.java

```diff
@@ -465,9 +465,7 @@ private boolean isHierarchicalNamespaceEnabled(AzureLocation location)
 
     private BlobClient createBlobClient(AzureLocation location)
     {
```

Comment:
`Such cases are covered by the TestAzureBlobFileSystem test.` 

Suggestion:

```
Such cases are covered by the test:

io.trino.tests.product.hive.TestAzureBlobFileSystem
```

Consider squashing this commit with the next commit.